### ISSUE TITLE
Docstring param description hidden when it contains a colon

### ISSUE THREAD

**Issue by bzamecnik** [google/python-fire#189]
Given a simple script:
```
import fire

def foo(name='World'):
    """
    Greets name.

    Arguments
    ---------
    name : str
        name, default: World
    """
    print('Hello %s' % name)

if __name__ == '__main__':
    fire.Fire(foo)
```

```
python foo.py -h
```

shows the help with the description of `--name` missing:

```
# ...
FLAGS
    --name=NAME
```

If the description does not contain a colon, like `name, default World` it works OK:

```
FLAGS
    --name=NAME
        name, default World
```

It should work even with a colon. I'd guess it's connected with parsing the `param_name : type` section which contains also a colon.

`fire==0.2.1`

**Comment by dbieber** [comment]
This will require making changes to https://github.com/google/python-fire/blob/master/fire/docstrings.py

The `if` at https://github.com/google/python-fire/blob/master/fire/docstrings.py#L450, and particularly the `elif` at https://github.com/google/python-fire/blob/master/fire/docstrings.py#L457 are the first placed I would look. This elif controls the behavior when a colon is encountered in a numpy args section.

**Comment by dbieber** [comment]
Fixed by #201 with b1f3f95d736e7e36126f2e6c457c52f995f587dc
Thanks Vincent!
